Sources/Kitura/SwaggerGenerator.swift

```diff
@@ -600,20 +614,25 @@ struct SwaggerDocument: Encodable {
 
             // Check to see if we have already built this model
             guard self.definitions[model] == nil else {
-                Log.debug("Already generated model \(model)")
+                Log.debug("Already generated model '\(model)'")
                 return
             }
 
             // then build all it
-            if let modelInfo = try? buildModel(typeInfo) {
-                Log.debug("in addModel(model: \(model))")
+            do {
+                Log.verbose("Building model: '\(model)'")
+                let modelInfo = try buildModel(typeInfo)
                 if modelInfo.required.count > 0 {
                     modelDefinition = SwaggerModel(type: "object", properties: modelInfo.properties, required: Array(modelInfo.required))
                 } else {
                     modelDefinition = SwaggerModel(type: "object", properties: modelInfo.properties, required: [])
                 }
                 self.definitions[model] = modelDefinition
+            } catch {
+                Log.error("Failed to build model '\(model)': \(error)")
             }
+        } else {
```

Comment:
Can we make the `if case` for this a `guard` so we can remove the indentation?

Reply:
This will become a switch in a future PR (we need to recursively add models that are nested within unkeyed / dynamicKeyed cases)